test/python_tests/sqlite_test.py

```diff
@@ -469,7 +469,7 @@ def test_that_64bit_int_fields_work():
     def test_null_id_field():
         # silence null key warning: https://github.com/mapnik/mapnik/issues/1889
         default_logging_severity = mapnik.logger.get_severity()
-        mapnik.logger.set_severity(mapnik.severity_type.None)
+        mapnik.logger.set_severity(getattr(mapnik.severity_type, "None"))
```

Comment:
What is the issue with None here? If this breaks Python 3 we should change the name of the variable?